wurst/systems/boards/PlayerScore.wurst

```diff
@@ -94,8 +95,8 @@ public class PlayerScore
             goldAmount += goldAdded
 
             // Updating gold amount for tribe player
-            // TODO: Disable if -tb isn't on?
-            playerScores.get(this._player.getTribe().getNonMember()).goldAmount += goldAdded
+            if this._player.getTribe() != null
```

Comment:
Same question as below. Is this just enforcing that the total for a tribe is the sum of the members? That should be computed retroactively if anything.